PyStemmusScope/forcing_io.py

```diff
@@ -164,19 +171,23 @@ def prepare_global_variables(data, input_path, config):
     utils.remove_dates_from_header(input_path / "forcing_globals.mat")
 
 
-def prepare_forcing(config):
-    """Function to prepare the forcing files required by STEMMUS_SCOPE. The input
-        directory should be taken from the model configuration file.
+def prepare_forcing(config, forcing_filename):
```

Comment:
The function `prepare_forcing` should be able to prepare data with information in `config` which has `Location` key. Instead of passing `forcing_filename` to this function, let's use `forcing_filename= utils.get_forcing_file(config)`, see my comment below.